docs/framework/interop/index.md

```diff
@@ -34,7 +34,7 @@ Describes marshaling for COM interop and platform invoke.
 [How to: Map HRESULTs and Exceptions](how-to-map-hresults-and-exceptions.md)  
 Describes the mapping between exceptions and HRESULTs.
 
-[COM Wrappers](com-wrappers.md)  
+[COM Wrappers](../../standard/native-interop/com-wrappers.md)  
```

Comment:
If this in standard folder, it should not be in the in this section anymore. Move to a Related sections perhaps?